engagement/1.x/README.md

```diff
@@ -1,10 +1,19 @@
 # Slickstream Engagement Suite Developer Documentation (v1.x)
 
-## Widget placeholders
+This documentation is to support Slickstream customers and their web developers who wish to integrate Slickstream's Engagement Suite more seamlessly into their website.
 
-When Slickstream adds a widget on a page, it looks for an element (usually a `<div>`) with an appropriate class name and then injects the widget as a child of that element. 
+Slickstream is primarily a client-side technology -- meaning that we add functionality into a website from inside the browser, rather than when the pages are rendered by the web server.
 
-Websites can take advantage of this by pre-allocating a space for the widget. Allocating a space reduces the [Cumulative Layout Shift (CLS)](https://web.dev/cls/) on the page. 
+There are three ways in which website developers can improve integration with Slickstream:
+
+- [Positioning](#positioning)
+- [Activating](#activating)
+- [Styling](#styling-slickstream-widgets)
+- [Javascript API](#slickstream-javascript-api-v1.0)
+
+## Positioning
+
+Slickstream injects widgets onto a page after the page has loaded.  We inject these widgets into a container (as a child element of that container).  The container is typically a <div> with a certain CSS class.  We are able to add that container itself if necessary.  But it is even better if the page already has the container (as rendered by the server) so that it can pre-allocate space for our widget.  Allocating space reduces the [Cumulative Layout Shift (CLS)](https://web.dev/cls/) on the page and lets you be as specific as you like about the positioning of the Slickstream widget.  Our widgets are responsive and will use the available space appropriately.
```

Comment:
I would put any code or values in back ticks. `<div>` rather than <div>. Thats the common style in these kinda docs.